quickstart/faqs.md

```diff
@@ -74,3 +74,11 @@ Mayastor, as any other solution leveraging TCP for network transport, may suffer
 ## Why do Mayastor pods show high levels of CPU utilization when there is little or no I/O being processed?
 
 Mayastor has been designed so as to be able to leverage the peformance capabilities of contemporary high-end solid-state storage devices.  A significant aspect of this is the selection of a polling based I/O service queue, rather than an interrupt driven one.  This minimises the latency introduced into the data path but at the cost of additional CPU utilisation by the "reactor" - the poller operating at the heart of the Mayastor pod.  When Mayastor pods have been deployed to a cluster, it is expected that these daemonset instances will make full utilization of their CPU allocation, even when there is no I/O load on the cluster.  This is simply the poller continuing to operate at full speed, waiting for I/O.  For the same reason, it is recommended that when configuring the CPU resource limits for the Mayastor daemonset, only full, not fractional, CPU limits are set; fractional allocations will also incur additional latency, resulting in a reduction in overall performance potential. The extent to which this performance degradation is noticeable in practice will depend on the performance of the underlying storage in use, as well as whatvever other bottlenecks/constraints may be present in the system as cofigured.
+
+
+## What happens when a PV with reclaim policy set to `retain` is deleted?
+
+When the reclaim policy is set to 'retain`, the underlying volume remains intact even after deletion of the PV, which leaves the volumes unusable. In this scenario, the external provisioner does not call the delete volume API. To overcome this, Mayastor deploys a PV garbage collector as part of the CSI controller. The garbage collector deletes the underlying volume object associated with the deleted PV with reclaim policy set to retain. 
```

Comment:
Start with a short description (one-line definition) of "retain" first. Then we can continue with "When the reclaim policy..."